### ISSUE TITLE
[Feature Request] Option to remove or add columns & rows in the application library

### ISSUE THREAD

**Issue by senntore** [pop-os/cosmic-applibrary#83]
I try to be more organized with the application library, so I've separated most of the applications I use in related folders. This results in having  a lot of empty spaces in folders & even in Library Home.
I think Pop OS 22.04 has 7 columns & 3 rows by default & prefer that. But since the new Cosmic has 7 columns & 4 rows, the application library covers much more of the screen & most of it is empty(unless I'm in Utilities folder/group).

I would love if you can add an option to add or remove columns & rows with minimum of only 3 rows.

Note: The experience is from a 1920 x 1080 screen, so it might be different for other resolution.

**Comment by git-f0x** [comment]
The mentioned issue seems to have been fixed (it's now also 7 columns and 3 rows in COSMIC), but the feature request isn't implemented. I guess it could be useful at higher resolutions or if often opening it on a portrait display. Though I'm not sure what the best place to configure it would be, if implemented.

**Comment by senntore** [comment]
That's true! You could add the settings in Desktop and Panel, as it is quite empty & that is the only place App Library is mentioned.
Or you could add the settings in it's applets, as settings are planned for applets in Alpha 2.
